starter/cms-sanity-graphql-fragments/src/app/[name]/page.tsx

```diff
@@ -0,0 +1,135 @@
+import { PortableText } from "@portabletext/react";
+import { registerUrql } from "@urql/next/rsc";
+import type { Metadata } from "next";
+import Link from "next/link";
+import { notFound } from "next/navigation";
+import { cache } from "react";
+import { createGraphQLClient, graphql } from "@/lib/graphql";
+
+export const dynamic = "force-static";
```

Comment:
I'd prefer replacing this with `generateStaticParams()` to allow for ISR, even with an empty array it'll be static.